auslib/test/web/test_client.py

```diff
@@ -442,6 +442,95 @@ def setUp(self):
     }
 }
 """)
+        dbo.rules.t.insert().execute(priority=200, backgroundRate=100,
+                                     mapping='superblobaddon-with-multiple-response-blob', update_type='minor',
+                                     product='superblobaddon-with-multiple-response-blob',
+                                     data_version=1)
+        dbo.rules.t.insert().execute(priority=200, backgroundRate=100,
+                                     mapping='superblobaddon-with-one-response-blob', update_type='minor',
+                                     product='superblobaddon-with-one-response-blob',
```

Comment:
I can see why you were confused about blob name and product now, since we often use the same value in these tests. It might help to have a look at http://localhost:8080/releases once you get your Docker images working again. You'll notice that there are only a few different product names, but many blobs in each product.

You're just following the current style here, so need to change anything in these tests.